node/external/nodeApiResolver.go

```diff
@@ -21,9 +26,9 @@ func NewNodeApiResolver(scDataGetter ScDataGetter, statusMetricsHandler StatusMe
 	}, nil
 }
 
-// GetVmValue retrieves data stored in a SC account through a VM
-func (nar *NodeApiResolver) GetVmValue(address string, funcName string, argsBuff ...[]byte) ([]byte, error) {
-	return nar.scDataGetter.Get([]byte(address), funcName, argsBuff...)
+// SimulateRunSmartContractFunction retrieves data stored in a SC account through a VM
```

Comment:
Rename to `RunSmartContractFunction`, it will be cleaner and more accurate.

Reply:
(see above, `ExecuteSCQuery`)